content/en/docs/quick-start/minimal-kubesphere-on-k8s.md

```diff
@@ -1,8 +1,66 @@
 ---
 title: "Minimal KubeSphere on Kubernetes"
-keywords: 'kubesphere, kubernetes, docker, multi-tenant'
-description: 'Install a Minimal KubeSphere on Kubernetes'
+keywords: 'KubeSphere, Kubernetes, Minimal, Installation'
+description: 'Minimal Installation of KubeSphere on Kubernetes'
 
 linkTitle: "Minimal KubeSphere on Kubernetes"
 weight: 3020
 ---
+
+In addition to installing KubeSphere on a Linux machine, you can also deploy it on existing Kubernetes clusters directly. This QuickStart guide walks you through the general steps of completing a minimal KubeSphere installation on Kubernetes. For more information, see [Installing on Kubernetes](https://kubesphere-v3.netlify.app/docs/installing-on-kubernetes/).
+
+## Prerequisites
+
+Below is a quick check of your environment. For more information, see [Prerequisites](https://kubesphere-v3.netlify.app/docs/installing-on-kubernetes/introduction/prerequisites/).
+
+- Kubernetes version: `1.15.x, 1.16.x, 1.17.x, 1.18.x`;
+- CPU > 1 Core; Memory > 2 G;
+- A default Storage Class in your Kubernetes cluster is configured; use `kubectl get sc` to verify it.
+- The CSR signing feature is activated in kube-apiserver when it is started with the `--cluster-signing-cert-file` and `--cluster-signing-key-file` parameters. See [RKE installation issue](https://github.com/kubesphere/kubesphere/issues/1925#issuecomment-591698309).
+
```

Comment:
Please also add a `Warning` for the listed prerequisites. Then add a prompt for metrics-server.